packages/flutter/test/material/date_range_picker_test.dart

```diff
@@ -252,19 +251,12 @@ void main() {
       await tester.pumpAndSettle();
     });
 
-    testWidgetsWithLeakTracking('landscape', (WidgetTester tester) async {
+    testWidgets('landscape', (WidgetTester tester) async {
       await showPicker(tester, kCommonScreenSizeLandscape);
       expect(tester.widget<Text>(find.text('Jan 15 – Jan 25, 2016')).style?.fontSize, 24);
       await tester.tap(find.text('Cancel'));
       await tester.pumpAndSettle();
-    },
-      // TODO(polina-c): remove after resolving
-      // https://github.com/flutter/flutter/issues/130354
-      leakTrackingTestConfig: const LeakTrackingTestConfig(
-        allowAllNotGCed: true,
-        allowAllNotDisposed: true,
-      ),
-    );
+    });
```

Comment:
we should keep this todo with a new issue or reopen the existing issue if it is not fixed properly

Reply:
We have a big non-written 'TODO' for each 'testWidgets', to be converted.

I feel this exact one is failing because of issue already covered by TODO for other test or by non-merged fixing PR. When we fix that other TODO, this one and all others in this lib, most likely will convert smoothly.

If no, we will do deeper investigation.

I just want to save time on investigation for this now, as it seems it will not be needed. 
How does it sound?